### PR TITLE
Add "Total size" of Partition within Topic's statistics #2536

### PR DESCRIPTION
**Pull request by David-DB88**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Added total size in Partition within Topic's statistics 

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by workshur — APPROVED** [review]

**Comment by workshur** [comment]
@Haarolean pls take a look 👀

**Review by Haarolean — CHANGES_REQUESTED** [review]
image
Please make it human-readable just like the other sizes

File: kafka-ui-react-app/src/components/Topics/Topic/Statistics/PartitionInfoRow.tsx [hunk 1]
```diff
@@ -27,14 +27,15 @@ const PartitionInfoRow: React.FC<{ row: Row<TopicAnalysisStats> }> = ({
...
-
   return (
     <S.PartitionInfo>
       <div>
         <Heading level={4}>Partition stats</Heading>
         <List>
           <Label>Total message</Label>
           <span>{totalMsgs}</span>
+          <Label>Total size</Label>
+          <span>{(keySize?.sum || 0) + (valueSize?.sum || 0)}</span>
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Statistics/PartitionInfoRow.tsx:38]: pls use BytesFormatted here

Comment (David-DB88) [reply kafka-ui-react-app/src/components/Topics/Topic/Statistics/PartitionInfoRow.tsx]: done

**Review by workshur — APPROVED** [review]
Looks good to me. @Haarolean please take a look again

**Review by Haarolean — APPROVED** [review]

**Pull request merged**